cl/custom_filters/templatetags/extras.py

```diff
@@ -1,4 +1,5 @@
 import random
+import re
```

Comment:
Since I'm doing one more look through here, we don't need this import anymore either, right?

Reply:
Looks like you did the remove

Reply:
I don't know what you're looking at, but it's still there.